pkg/libs/endpointaccessible/endpoint_accessible_controller.go

```diff
@@ -72,17 +79,6 @@ func (c *endpointAccessibleController) sync(ctx context.Context, syncCtx factory
 			defer cancel()
```

Comment:
> reqCtx, cancel := context.WithTimeout(ctx, 10*time.Second) // avoid waiting forever

does setting the request timeout has any effect since you are setting the timeout on the HTTP client (Timeout: 5 * time.Second)

Reply:
I guess not much, I guess just making sure? Perhaps @vareti would know why he did that, but we may never know